### PR TITLE
feat(card): add pf-m-display-lg and pf-m-rounded

### LINKED ISSUE #5334: Large card variant
Add a new variant of the card component per https://github.com/patternfly/patternfly/pull/3793 . New card has increased spacing for marketing content (Large card)

### PR DESCRIPTION
**Pull request by redallen**
**What**: Closes #5334

**Additional issues**: I made an interactive modifiers example to showcase it

### CODE REVIEW

**Review by KKoukiou — APPROVED** [review]
Nice parametrization of the examples code @redallen!

Your change it of course totally fine codewise, I just wonder why PF wants to expose to its consumers that isRounded property. It causes a really subtle difference, and I feel that the PF designers should decide if the cards should have rounded corners or not and not let the consumers make decisions about it.

After all, the reason that we all use PF and not custom styling is in order that all RH products have the same Ui experience.

So, this PR is fine to go in but I really don't see the usecase for it.

**Comment by redallen** [comment]
@KKoukiou This thread has the context behind pf-m-rounded: https://github.com/patternfly/patternfly/issues/3670 . I'd ask @mcarrano its purpose.

**Comment by mcarrano** [comment]
You ask a valid question @KKoukiou . This originated from the web properties design team wanting to use PatternFly to present more marketing oriented content. @mmenestr do you know if we have plans to add this guidance to our design guidelines for Cards? I agree that we don't want designers to just arbitrarily pick styling. Products and web applications should stick with the squared corners.

**Comment by mmenestr** [comment]
@mcarrano I don't believe we do. This is the first I've heard of this! Just to make sure I have the info right, is it just a modifier to give cards the capability of having rounded edges? And we want to recommend that this only be done for marketing purposes and that products keep using non-rounded cards?

**Review by mcoker — COMMENTED** [review]
`isLarge` and `isCompact` shouldn't be used together - do we need to do anything special for that?

**Comment by redallen** [comment]
@mcoker Unfortunately since we've already released `isCompact` there isn't much we can do. I've added a warning and will default to `isCompact` if both are specified.

**Comment by kmcfaul** [comment]
Can we also add something to both prop descriptions (isLarge and isCompact) for clarity? Like `Should not be used together with {X}`. Or maybe the console warning is enough?

**Comment by redallen** [comment]
@kmcfaul I added that to the prop description.

**Review by mcoker — APPROVED** [review]
L👍TM!

**Review by jessiehuff — COMMENTED** [review]
I really like the modifier example! It's really neat to be able to play with all of the different modifiers. Can we add the keyboard interaction to the selectable example though? I currently can't select it by keyboard or screen reader the way I could with the previous selectable example. (I'm hoping that by adding the keyboard interaction, it'll work through screen reader as well.)

**Comment by redallen** [comment]
@jessiehuff I've removed the `isSelectable` and `isSelected` to avoid confusion and keep the demo simple.

**Review by mcarrano — DISMISSED** [review]
Looks good to me. Thanks @redallen .

File: packages/react-core/src/components/Card/Card.tsx [hunk 1]
```diff
@@ -47,12 +51,19 @@ export const Card: React.FunctionComponent<CardProps> = ({
...
+  isRounded = false,
+  isLarge = false,
   ouiaId,
   ouiaSafe = true,
   ...props
 }: CardProps) => {
   const Component = component as any;
   const ouiaProps = useOUIAProps(Card.displayName, ouiaId, ouiaSafe);
+  if (isCompact && isLarge) {
+    // eslint-disable-next-line no-console
```
Comment (tlabaj) [inline packages/react-core/src/components/Card/Card.tsx:63]: can you add a test for this

**Review by tlabaj — CHANGES_REQUESTED** [review]
looks good.  Can you just update the demo app and add test for new props.

File: packages/react-core/src/components/Card/Card.tsx (hunk 1 above)
Comment (redallen) [reply packages/react-core/src/components/Card/Card.tsx:63]: Done

**Review by kmcfaul — APPROVED** [review]

**Pull request merged**